src/main/resources/edu/cornell/kfs/module/ld/cu-spring-ld.xml

```diff
@@ -95,5 +95,16 @@
    <import resource="document/validation/configuration/SalaryExpenseTransferValidations.xml" />
    <import resource="document/validation/configuration/CuYearEndBenefitExpenseTransferValidations.xml" />
    <import resource="document/validation/configuration/CuYearEndSalaryExpenseTransferValidations.xml" />
+   
+   <!-- CU Customization KFSPTS-34828 
+   partial backport of FINP-10395 to fix opening general ledger from salary expense transfer document -->
+   <bean id="laborLedgerEntryForExpenseTransferLookupableHelperService"
+          parent="laborLedgerEntryForExpenseTransferLookupableHelperService-parentBean"
+          scope="prototype"/>
+    <bean abstract="true"
+          class="org.kuali.kfs.module.ld.businessobject.lookup.LedgerEntryForExpenseTransferLookupableHelperServiceImpl"
+          id="laborLedgerEntryForExpenseTransferLookupableHelperService-parentBean"
+          parent="laborLedgerEntryLookupableHelperService"
```

Comment:
One of the subsequent FINP-10395 commits changed this bean's parent to "laborLedgerEntryLookupableHelperService-parentBean" instead. You may want to update this bean override to do the same.